dart/lib/src/sentry.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:meta/meta.dart';
+
+import 'client.dart';
+import 'protocol.dart';
+import 'sentry_options.dart';
+import 'stack_trace.dart';
+
+/// Sentry SDK main entry point
+///
+class Sentry {
+  static SentryClient _client;
+
+  Sentry._();
+
+  static void init(SentryOptions options) {
+    _client = SentryClient(
+      dsn: options.dsn,
+      environmentAttributes: options.environmentAttributes,
+      compressPayload: options.compressPayload,
+      httpClient: options.httpClient,
+      clock: options.clock,
+      uuidGenerator: options.uuidGenerator,
+    );
+  }
+
+  /// Initializes the SDK
+  static void initDns(String dns) => _client = SentryClient(dsn: dns);
```

Comment:
we don't have this on other SDKs

Reply:
I removed it, but now the only way to init Sentry is to define an `OptionsConfiguration<SentryOptions>` which IMHO is not the simplest way for a beginner.

Reply:
our docs state that and all the SDKs should look like this, also the samples, it should not be an issue